integrations/woopra/lib/index.js

```diff
@@ -22,14 +22,11 @@ var Woopra = (module.exports = integration('Woopra')
   .option('cookieName', 'wooTracker')
   .option('cookieDomain', null)
   .option('cookiePath', '/')
-  .option('ping', true)
-  .option('pingInterval', 12000)
   .option('idleTimeout', 300000)
-  .option('downloadTracking', true)
-  .option('outgoingTracking', true)
```

Comment:
Changing the default behavior for `downloadTracking` and `outgoingTracking` would be a breaking change for existing users of this integration. Can these be reverted back to `true` by default or does it cause issues with `clickTracking`?

Reply:
The default configuration of the JS tracker is false for both these features. When it was true, Segment was assuming that those features are already enabled by default and was not including them in the injected javascript configuration. So switching the default to false fixes this issue (if my understanding is correct).

Reply:
We're actually calling `config` on each option when we initialize the integration:
https://github.com/segmentio/analytics.js-integrations/blob/61bd44991705cf4f07937a626cb70cb8aec2404c/integrations/woopra/lib/index.js#L77-L82
So the current behavior would be that these are both set to `true` by default.

Reply:
Thanks for the clarification. I was under the impression that Segment is only sending a config update for properties that do not match the default setting. I'll switch those to true.